tests/parsers/agilent_tapestation_analysis/agilent_tapestation_analysis_structure_test.py

```diff
@@ -288,7 +281,7 @@ def testcreate_measurement_groups_with_regions() -> None:
             comment="Partially Degraded",
         ),
         ProcessedDataFeature(
-            identifier="dummy_id",
+            identifier="TEST_ID_6",
```

Comment:
is there a rhyme or reason behind the TEST_ID_X? It does seem random

Reply:
The incrementing ID is really just a way to make the fake uuids distinct. We could instead use a consistent seed for uuid generation, but I think it would be harder to trace around.

For what it's worth, if you're familiar with a particular parser, the IDs usually make sense to you.

In this, for example, the parser generates all the processed data and calculated data items for a particular measurement, then moves to the next one.

That's why we get processed id 3, then calculated ids 4 & 5, then processed id 6.